cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/io/cdap/cdap/etl/common/Constants.java

```diff
@@ -74,6 +74,21 @@ private Constants() {
     public static final String AGG_GROUPS = "aggregator.groups";
     public static final String JOIN_KEYS = "joiner.keys";
     public static final String DRAFT_COUNT = "draft.count";
+
+    public static final class Connection {
+      public static final String CONNECTION_COUNT = "connection.count";
+      public static final String CONNECTION_TYPE_COUNT_FORMAT = "connection.%s.count";
```

Comment:
instead of exposing the format string directly, it would be a little cleaner to expose a method that takes the connection type and outputs the right connection name. That way there is no possibility of accidentally using the string without replacing the '%s'. Also, we'll be less likely to break things if we change the name in the future.